docs/designers-developers/developers/block-api/README.md

```diff
@@ -4,8 +4,8 @@ Blocks are the fundamental element of the Gutenberg editor. They are the primary
 
 ## Registering a block
 
-All blocks must be registered before they can be used in the editor. You can learn about block registration, and the available options, in the [block registration](block-api/block-registration.md) documentation.
+All blocks must be registered before they can be used in the editor. You can learn about block registration, and the available options, in the [block registration](../../docs/designers-developers/developers/block-api/block-registration.md) documentation.
```

Comment:
Re-looking at this in the sober light of day and I realised the paths aren't deep enough. Actually needs to go up 4 levels, like so:

```suggestion
All blocks must be registered before they can be used in the editor. You can learn about block registration, and the available options, in the [block registration](../../../../docs/designers-developers/developers/block-api/block-registration.md) documentation.
```